php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -166,10 +167,10 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
 
     $cmd_path = implode( ' ', get_path( $this ) );
     foreach ( $validator->get_unknown() as $token ) {
-      \Terminus::warning( sprintf(
-        "The `%s` command has an invalid synopsis part: %s",
-        $cmd_path, $token
-      ) );
+      \Terminus::get_logger()->warning(
```

Comment:
I thought we were targeting 5.5+ for the 1.0 release and i believe that this sort of chaiging is only a problem in PHP 4. 

Either way, I'm not super happy with these remaining static access calls. It's too tight a coupling, and opens the gates to leaking output a bit. I'll see what I can do about getting rid of them properly.